helpers/helpers.go

```diff
@@ -0,0 +1,50 @@
+package helpers
```

Comment:
this file contains helpers to convert go-algorand types to sdk types. we should be able to remove this file once all types have been converted to sdk types.

Reply:
Good idea. Worth adding as a comment/todo
```suggestion
package helpers

// this file contains helpers to convert go-algorand types to sdk types.
// TODO: remove this file once all types have been converted to sdk types.
```